Assets/MixedRealityToolkit/UX/Scripts/Buttons/Utilities/CompoundButtonToggle.cs

```diff
@@ -112,11 +112,8 @@ private void OnDisable()
 
             // Get the profile field of the target component and set it to the on profile
             // Store all icons in iconLookup via reflection
-#if USE_WINRT
-            FieldInfo fieldInfo = Target.GetType().GetTypeInfo().GetField("Profile");
```

Comment:
Whereas I didn't like these blocks everywhere for reflection. Is this flattening of the lookup going to cause future issues?

Reply:
See https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/1666#discussion_r163919711

Reply:
`USE_WINRT` was defined in a different file and not included in our builds, so this line was never called. Replacing it with what `USE_WINRT` was previously defined as caused a build error, which leads me to believe this line of code hasn't been called in a while haha.

![image](https://user-images.githubusercontent.com/3580640/35420670-c31a2096-01f3-11e8-9700-8601dce4dea0.png)

Reply:
Ahh, I thought it was from the reflection extensions

Reply:
Yes, I had to fix those errors, which only shows up on a UWP build.
Fixed by altering it's name accordingly.